services/py-api/src/service/hackathon_service.py

```diff
@@ -53,6 +53,20 @@ async def create_participant_and_team_in_transaction(
             self._create_participant_and_team_in_transaction_callback, input_data
         )
 
+    async def create_random_participant(
```

Comment:
What's the purpose of this method? It's doing literally the same thing `self._participant_repo.create` does.